tests/src/lib.rs

```diff
@@ -207,3 +207,17 @@ async fn ws_more_request_than_buffer_should_not_deadlock() {
 		req.await.unwrap();
 	}
 }
+
+#[tokio::test]
+async fn https_works() {
+	let client = HttpClient::new("https://kusama-rpc.polkadot.io", HttpConfig::default()).unwrap();
```

Comment:
I don't mind this actually, but some would make a big stink about having tests depend on the network.
@maciejhirsz do you have a strong opinion here (if not, then let's keep it as-is)?

Reply:
well, I don't know how to test it locally, I suppose I need to create a self-signed certificate on localhost? This was simpler :)

Any suggestions?